projects/aws/eks-a-snow-admin-ami/provisioners/cleanup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+sudo apt-get -y autoremove
+sudo apt-get -y clean
+sudo rm -rf /tmp/*
+cat /dev/null > ~/.bash_history
+history -c
```

Comment:
There is some additional stuff the image-builder does to cleanup the ami before exporting, https://github.com/kubernetes-sigs/image-builder/blob/master/images/capi/ansible/roles/sysprep/tasks/main.yml#L39

I know the machine-id is important on the node image, not sure how critical it is for the admin ami/on aws, but may be worth looking through this list to see if we need any here.

Reply:
Added a bunch of stuff from there